packages/core/test/oneOf_test.js

```diff
@@ -454,7 +454,7 @@ describe("oneOf", () => {
     expect(node.querySelector("select").value).eql("1");
   });
 
-  it("should not change the selected option when entering values on a subschema with multiple required options", () => {
+  it.only("should not change the selected option when entering values on a subschema with multiple required options", () => {
```

Comment:
Do the other tests pass?
```suggestion
  it("should not change the selected option when entering values on a subschema with multiple required options", () => {
```

Reply:
Yikes... another good catch, I'll find out in a few minutes ;)